src/olympia/abuse/tasks.py

```diff
@@ -43,31 +40,13 @@ def flag_high_abuse_reports_addons_according_to_review_tier():
 
     tier_filters = Q()
     for usage_tier in usage_tiers:
-        tier_filters |= Q(
-            average_daily_users__gte=usage_tier.lower_adu_threshold,
-            average_daily_users__lt=usage_tier.upper_adu_threshold,
-            abuse_reports_count__gte=F('average_daily_users')
-            * usage_tier.abuse_reports_ratio_threshold_before_flagging
-            / 100,
-        )
+        tier_filters |= usage_tier.get_abuse_threshold_q_object()
     if not tier_filters:
         return
 
-    abuse_reports_count_qs = (
-        AbuseReport.objects.values('guid')
-        .filter(
-            ~AbuseReportManager.is_individually_actionable_q(),
-            guid=OuterRef('guid'),
-            created__gte=datetime.now() - timedelta(days=14),
-        )
-        .annotate(guid_abuse_reports_count=Count('*'))
-        .values('guid_abuse_reports_count')
-        .order_by()
-    )
     qs = (
-        Addon.unfiltered.exclude(status=amo.STATUS_DISABLED)
-        .filter(type=amo.ADDON_EXTENSION)
-        .annotate(abuse_reports_count=Subquery(abuse_reports_count_qs))
+        UsageTier.get_base_addons()
+        .alias(abuse_reports_count=UsageTier.get_abuse_count_subquery())
```

Comment:
how does `alias` differ in functionality to `annotate` here?

Reply:
Super minor but `alias()` doesn't end up in the fields returned by the query. So for annotations that are only useful as something to `filter()` or `exclude()` but that you don't need the value later, it's slightly more efficient.